src/Core/System/Test/Snippet/Mock/_fixtures/storefront.de-DE.json

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
This file seems to be still in use by 'platform/src/Core/System/Test/Snippet/Files/SnippetFileCollectionTest.php', could you please restore it? (I guess that was deleted by some tool? Else please also mention these things on the changelog and PR description)

Reply:
I have no idea why the file got removed. I restored it.